vm_supervisor/__main__.py

```diff
@@ -270,7 +296,10 @@ def main():
         ALLOW_VM_NETWORKING=args.allow_vm_networking,
         FAKE_DATA_PROGRAM=args.fake_data_program,
         DEBUG_ASYNCIO=args.debug_asyncio,
+        FAKE_INSTANCE_BASE=args.fake_instance_base,
     )
+    if args.run_fake_instance:
+        settings.USE_FAKE_INSTANCE_BASE = True
```

Comment:
```suggestion
        settings.FAKE_INSTANCE_BASE = True
```
You need to use the same setting name here: https://github.com/aleph-im/aleph-vm/blob/hoh-extend-instance-run/vm_supervisor/storage.py#L157

Reply:
I think that it would be better to have two settings:
`USE_FAKE_INSTANCE_BASE: bool` is a boolean that defines if the instance base from the message should be used or if a local file (rootfs.ext4) should be used as base.
`FAKE_INSTANCE_BASE` is the path to that rootfs base file, and has a sensible default for development and test purposes.